app/views/ops/_rbac_group_details.html.haml

```diff
@@ -70,8 +70,8 @@
             = link_to(@group.miq_user_role.name, "#", params)
         - else
           = select_tag('group_role',
-              options_for_select(@edit[:roles].sort, @edit[:new][:role]),
-              :class  => "selectpicker")
+                       options_for_select(@edit[:roles].sort, selected: @edit[:new][:role], disabled: ["<Choose a Role>", nil]),
```

Comment:
@hstastna use arrows instead of the colon for `selected` and `disabled` parameters

Reply:
@romanblanco Done ;)